src/features/layout/PageNotFound.tsx

```diff
@@ -1,11 +1,23 @@
-import { FC } from 'react';
+import { usePageTitle } from './layoutSlice';
 
-const PageNotFound: FC = () => (
-  <>
+export const TITLE = 'Page Not Found';
+
+export interface PageNotFoundProps {
+  message?: string;
+}
+const PageNotFound = ({ message }: PageNotFoundProps) => {
+  function renderMessage() {
+    if (message) {
+      return <p>{message}</p>;
+    }
+  }
+  usePageTitle(TITLE);
+  return (
     <section>
-      <h2>Page Not Found.</h2>
+      <h2>{TITLE}</h2>
+      {renderMessage()}
```

Comment:
You mean `renderMessage()`? Yeah, I make liberal use of internal render functions/methods so that jsx markup doesn't get out of hand with inline expressions, etc.

Reply:
Indeed, I am pretty used to inlining {blah?foo:bar} into react from my last job but this is cleaner in that situation.